src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -74,6 +76,10 @@
 
 namespace grpc_core {
 
+namespace {
```

Comment:
No need for an anonymous namespace around a `using` statement.  If you're not defining any symbols, then the anonymous namespace doesn't actually hide anything.

Reply:
Done.